packages/point-intercept/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@pie-ui/point-intercept",
+  "publishConfig": {
+    "access": "public"
+  },
+  "version": "0.0.1",
+  "description": "An interaction to allow students identify coordinates or plot points on a graph by clicking on the graph.",
+  "main": "src/index.js",
+  "dependencies": {
+    "@pie-lib/charting": "../../../pie-lib/packages/charting",
```

Comment:
This is until pie-lib gets updated with the respective changes that I'm using here.

Reply:
👍